edsnlp/pipes/ner/behaviors/alcohol/patterns.py

```diff
@@ -3,7 +3,7 @@
     regex=[
         r"\balco[ol]",
         r"\bethyl",
-        r"(?<!(25.?)|(sevrage)).?\boh\b",
+        r"(?<!(25.{0,10}))\boh\b",
```

Comment:
Doesn't this add false positives ?

Reply:
Apparently no from the research project that suggested this change